3sum.java

```diff
@@ -0,0 +1,22 @@
+class Solution {
```

Comment:
Kindly mention time and space complexity as well . Kindly try to give optimized solution.